gui.cpp

```diff
@@ -785,7 +785,7 @@ void UI::minimize(tgui::ChildWindow::Ptr win) {
 
 void UI::changeServerAdd(sf::String addr) { //Quickfix for issue #33
 	std::string serveraddr = addr.toAnsiString();
-	if (serveraddr.back() != '.' || serveraddr.size() != 0)
+	if (!(serveraddr.back() == '.' || serveraddr.size() == 0))
```

Comment:
I think this is here because the pull-request was started before I made that change in the repo. Hence it shows up as a diff in the commits.
I'm not sure though... but it's actually changing it to what is already in the repo (if that makes any sense)